### PR TITLE
added Smith College

### PR DESCRIPTION
**Pull request by beanumber**

### CODE REVIEW

**Comment by ismayc** [comment]
Nice! Could you rename it something different than {thesisdown}, maybe {smithdown} or {pioneerdown}? I'd hate for there to be confusion down the road. I recommend digging into the template LaTeX files to replace "Reed College" with "Smith College" too at the very least. Let me know if I can assist!

**Comment by beanumber** [comment]
OK, I think this is better. Maybe @rudeboybert or @ambertin can check my work. 

Question: wouldn't it be easier for `pioneerdown::thesis_pdf` to just be a re-import from `thesisdown::thesis_pdf`? I don't anticipate making any changes to these functions, but would appreciate the upstream support.

**Comment by ismayc** [comment]
Yep! No need to make your own output and can just use the {thesisdown} output formats. I'll wait for confirmation through this weekend before I merge just in case any other edits are requested.

File: inst/rmarkdown/templates/thesis/skeleton/skeleton.Rmd [hunk 1]
```diff
@@ -2,28 +2,28 @@
...
 author: 'Your R. Name'
 date: 'May 20xx'
-institution: 'Reed College'
+institution: 'Smith College'
 division: 'Mathematics and Natural Sciences'
 advisor: 'Advisor F. Name'
 # If you have more two advisors, un-silence line 7
 #altadvisor: 'Your Other Advisor'
-department: 'Mathematics'
+department: 'Statistical & Data Scieces'
```
Comment (ismayc) [inline inst/rmarkdown/templates/thesis/skeleton/skeleton.Rmd:10]:
```suggestion
department: 'Statistical & Data Sciences'
```

**Comment by ismayc** [comment]
This all looks good to me. I'm gonna close this PR since it would also update {thesisdown} but I have added in your repo to the {thesisdown} README. This PR is incredibly helpful as well in that I could potentially add in a function to the {thesisdown} package to go through and make the updates that you have done here. Then someone could just apply the function and potentially create the {thesisdown} README update with just an R function call. This will be particularly valuable if their institution doesn't have their own LaTeX template. Thanks much, @beanumber!

**Pull request closed without merging**